src/elm/View/Messages.elm

```diff
@@ -100,9 +119,40 @@ viewMessage messageIndex message isLast =
     ]
 
 
-fadeInDelay : Int -> Int
-fadeInDelay messageIndex =
-    3000 * messageIndex
+fadeInDelayBasedOnMessageLength : Int -> Int -> Int -> Int
```

Comment:
Sometimes worth passing in a record when getting to 3 args of the same type to avoid accidentally reversing them, but I think in this case - fine since unlikely to make future changes here.